includes/class-wc-stripe-webhook-handler.php

```diff
@@ -768,6 +770,25 @@ public function is_partial_capture( $notification ) {
 		return 0 < $notification->data->object->amount_refunded;
 	}
 
+	/**
+	 * Gets the first refund object from charge notification.
+	 *
+	 * @since x.x.x
+	 * @param object $notification
+	 *
+	 * @return object
+	 */
+	public function get_refund_object( $notification ) {
+		// Since API version 2022-11-15, the Charge object no longer expands `refunds` by default.
+		// We can remove this once we drop support for API versions prior to 2022-11-15.
+		if ( ! empty( $notification->data->object->refunds->data[0] ) ) {
+			return $notification->data->object->refunds->data[0];
+		}
+
+		$charge = $this->get_charge_object( $notification->data->object->id, [ 'expand' => [ 'refunds' ] ] );
```

Comment:
The `'expand' => [ 'refunds' ]` parameter here isn't required while we're still using version `2019-09-09` from our requests to Stripe [here](https://github.com/woocommerce/woocommerce-gateway-stripe/blob/trunk/includes/class-wc-stripe-api.php#L17).

The current version will automatically expand refunds, but once we move to `2022-11-15` or later, this is going to be ultimately required, so to prevent future issues, I think it's better to be explicit about expanding the refunds property now.